Functions/GeoTrack.ps1

```diff
@@ -116,12 +135,13 @@ Function Get-GoogleLocationHistoryKmlFileUri {
 
 Function Get-GoogleLocationHistoryKmlFile {
     [CmdletBinding()] param(
+        [Microsoft.PowerShell.Commands.WebRequestSession] $session,
         [DateTime] $DateTime,
-        [FileInfo] $outFile
+        [System.IO.FileInfo] $outFile

```

Comment:
Adding the namespace is great, I'm curious what triggered this? Was it not working for you without the prefix?  I ask because I want to know if there was something special in my environment that enabled it to work.